### DISCUSSION TITLE
Gatsby On Netlify Deploy Issue

### DISCUSSION THREAD

**Discussion by talmax1124** [gatsbyjs/gatsby#30776] [Help]
When using deploy on netlify, gives me error. I tried changing the api key in netlify to see if it was that but no luck. 

$ npm run build
11:51:04 PM: > gatsby-starter-delog@1.6.0 build /opt/build/repo
11:51:04 PM: > GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES=true gatsby build --log-pages
11:51:08 PM: success open and validate gatsby-configs - 0.077s
11:51:10 PM: error Error in "/opt/build/repo/node_modules/gatsby-plugin-algolia/gatsby-node.js": Invalid or unexpected token
11:51:10 PM: 
11:51:10 PM: 
11:51:10 PM:   Error: /opt/build/repo/node_modules/gatsby-plugin-algolia/gatsby-node.js:47
11:51:10 PM:     const client = algoliasearch(appId, apiKey, { timeout: 30_000 });
11:51:10 PM:                                                            ^^
11:51:10 PM:   SyntaxError: Invalid or unexpected token
11:51:10 PM:

**Comment by LekoArts** [comment]
You should open an issue at: https://github.com/algolia/gatsby-plugin-algolia
